markdown/ch8.markdown

```diff
@@ -1,6 +1,18 @@
 # Big Data in Little Laptop with Toolz
 
-Whenever I think too hard about streaming data analysis, my head hurts.
+> GRACIE: A knife? The guy's twelve feet tall!
+> JACK: Seven. Hey, don't worry, I think I can handle him.
+>
+> — Jack Burton, Big Trouble in Little China
+
+Although streaming is not really a SciPy feature, it is a way of writing code
```

Comment:
Suggested rewording to try and capture:
- Contrast between this not being a SciPy feature but why we are still talking about it
- Emphasize that it is not a way of writing code, but a way to deconstruct the problem

We dedicate this chapter to streaming computation which, while not a SciPy feature as such, is an important way of computing solutions on large datasets using resource constrained machines (such as laptops).